### ISSUE TITLE
Implement MQTT Switch

### ISSUE THREAD

**Issue by mion00** [unixorn/ha-mqtt-discovery#31]
**Is your feature request related to a problem? Please describe.**
Now that we have built a code foundation after #15, I suggest we add additional entities.
I am considering for instance the [MQTT Switch](https://www.home-assistant.io/integrations/switch.mqtt/). 
We can easily define an additional class in `sensors.py`, as already done for the existing binary sensor and sensor, but I have a question about the features we would like to expose to the user of this library.
The switch is similar to a _binary sensor_ since it has on and off states, but at the same time it is sort of a _bidirectional_ entity: its state can be changed both from our side and from Home Assistant, via the `command_topic`.

So the question is: should the `Switch` object we provide in this library act upon messages on this topic to reflect the commands coming from HA? This is a big design decision on our side, since it means we are not only limiting this library to _publishing_ state changes, but we are also acting on state changes coming from HA.

**Describe the solution you'd like**
Maybe we can consider registering a `Callable` object provided by the end-user as part of the `Switch` instance that gets called with the state change message, so that custom code can run when a command is sent from HA to change the switch state.

**Describe alternatives you've considered**
We only publish state changes to HA, so a user of the library need to manually subscribe to the `command_topic` with additional code in his application.
